utils/check_config_docstrings.py

```diff
@@ -85,7 +85,11 @@ def check_config_docstrings_have_checkpoints():
 
     if len(configs_without_checkpoint) > 0:
         message = "\n".join(sorted(configs_without_checkpoint))
-        raise ValueError(f"The following configurations don't contain any valid checkpoint:\n{message}")
+        raise ValueError(
+            f"The following configurations don't contain any valid checkpoint:\n{message}\n\n"
+            "Hint: ensure to include [foo/bar](https://huggingface.co/foo/bar) link pointing to "
+            "one of the models of this architecture in the docstring of the config classes listed above."
+        )
```

Comment:
I would prefer to rephrase it like

```
ensure to include a link pointing to one of the models of this architecture in the docstring of the config classes listed above. The link should have a format looks like [foo/bar](https://huggingface.co/foo/bar).
```

Reply:
Thank you, @ydshieh 

sure, I used what you suggested and further improved it I hope. Let me know if this is satisfactory.

Reply:
Way better, thanks a lot. Just a `s` thing in `requirements` maybe. Feel free to merge 🚀